core/src/components/button/button.tsx

```diff
@@ -2,11 +2,11 @@ import { Component, Element, h, Host, Prop, State } from '@stencil/core';
 import { disableClickOnElement } from '../../utils/utils';
 
 @Component({
-  tag: 'sdds-button',
+  tag: 'tds-button',
   styleUrl: 'button.scss',
   shadow: true,
 })
-export class SddsButton {
+export class TdsButton {
   @Element() host: HTMLElement;
 
   /** Text displayed inside the Button */
```

Comment:
Suggestion: As non of the comments were touched on props/methods etc readme file still contains #sdds-button on the top. You can trick Stencil to regenerate it by adding a typo or comma in one of those comments in this file and it will generate the correct readme, once it does, you can revert the typo.

Reply:
Actually did not work, so had to fix it manually.